src/Avalonia.Controls/SplitView.cs

```diff
@@ -169,7 +177,7 @@ static SplitView()
             PanePlacementProperty.Changed.AddClassHandler<SplitView>((x, v) => x.OnPanePlacementChanged(v));
             DisplayModeProperty.Changed.AddClassHandler<SplitView>((x, v) => x.OnDisplayModeChanged(v));
 
-            PaneProperty.Changed.AddClassHandler<SplitView>((x, e) => x.PaneChanged(e));
+            PaneProperty.Changed.AddClassHandler<SplitView>((x, e) => x.OnPaneChanged(e));
```

Comment:
I would like to remove all these separate methods and just use the OnPropertyChanged method which is more common and a bit faster now. Please let me know if there are objections to this refactor.

Reply:
You mean OnPanePlacementChanged, OnPaneChanged, OnUseLightDismissChanged, OnCompactPaneLengthChanged and OnDisplayModeChanged? No objections.

Reply:
Ok, went ahead and moved all to OnPropertyChanged